packages/react-scripts/config/webpack.config.ssr.js

```diff
@@ -100,16 +106,16 @@ module.exports = function(webpackEnv) {
     preProcessorOptions = {}
   ) => {
     const loaders = [
-      isEnvDevelopment && require.resolve('style-loader'),
-      isEnvProduction && {
-        loader: MiniCssExtractPlugin.loader,
-        options: Object.assign(
-          {},
-          shouldUseRelativeAssetPaths ? { publicPath: '../../' } : undefined
-        ),
-      },
+      // isEnvDevelopment && require.resolve('style-loader'),
```

Comment:
Could you confirm why this has been removed?

Reply:
In standard CRA, in development, styles are served client-side only, via the Webpack dev server. The CSS is served inside the `.js` assets.

I didn't want to get into changing that - so in development, SSR should just ignore styles completely.